src/main/java/mSearch/tool/MVHttpClient.java

```diff
@@ -0,0 +1,41 @@
+package mSearch.tool;
+
+import okhttp3.ConnectionPool;
+import okhttp3.OkHttpClient;
+
+import java.util.concurrent.TimeUnit;
+
+public class MVHttpClient {
+    private final static MVHttpClient ourInstance = new MVHttpClient();
+    //private static final Cache cache = new Cache(new File("./cache"), 1024L * 1024L * 200L);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)